sources/packages/forms/src/form-definitions/studentwelcomepage.json

```diff
@@ -148,7 +148,7 @@
                       "value": ""
                     }
                   ],
-                  "content": "Your welcome page features helpful links which will allow you to start an application or manage your loan.",
+                  "content": "Welcome to your Student Account!\n<br><br>\nStudentAid BC helps eligible students with the cost of post-secondary education through student loans, grants, and other financial assistance programs. From this page you can:\n<ul>\n<li>Apply for student financial assistance.</li>\n<li>View your current and past applications and assessments</li>\n<li>Track your application status</li>\n<li>Upload and view documents</li>\n<li>Request a change to an active application</li>\n<li>Manage your StudentAid BC profile</li>\n<li>View your account activity</li>\n</ul>",
```

Comment:
Please use the custom class `align-bullets` in the custom class section to ensure padding to the bullets.

![image](https://github.com/user-attachments/assets/8bea1fc2-5f9c-44d3-9b8a-0e2676077948)

![image](https://github.com/user-attachments/assets/23a6a96e-8418-4e72-a1c5-06333c22f8ec)

Reply:
updated:
<img width="1242" alt="image" src="https://github.com/user-attachments/assets/88ddb013-6ab1-4959-ae70-e3460a44d0ae">